.github/workflows/ci.yml

```diff
@@ -43,6 +43,20 @@ jobs:
       - name: Run Cargo.toml linter
         run: git ls-files | grep Cargo.toml$ | xargs --verbose -n 1 cargo-toml-lint
 
+  check-rustc:
+    needs:
+      - cancel-previous-runs
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check rustc version
+        run: |
+          # Check rustc version, install if not on latest stable
+          RUSTC_VERSION=$(rustc --version | awk '{print $2}')
```

Comment:
@rvmelkonian 
- There's no one way to do this but here are some thoughts (I see the issue is skimpy on the details 😅 )
- Context
  - We had a situation not too long ago where we were 2 releases behind the latest stable `rustc`
  - We've only been learning about new `rustc` releases randomly (whether online or by checking the `#rust-sdk` team Slack channel)
  - Ideally we want this little CI job to just tell us whether we're on the latest `rustc` or not. A 👍  or 👎 
    - What happens when we're _not_ on the latest `rustc` is TBD - but I imagine we'd just make a PR then and there to upgrade to the latest
    - Since other jobs (e.g., `get-workspace-members`) would be dependent on this this `check-rustc` job, if `check-rustc` fails that's all we need to know/do 👌 
- What does that look like in practice?
  - There's obviously a lot of ways to do this - but maybe the simplest would be to hardcode ([here](https://github.com/FuelLabs/fuel-indexer/blob/master/.github/workflows/ci.yml#L11)) which version of Rust we (internal contributors) are on and then compare the output of what you have (`rustc --version | awk '{print $2}'`)

@deekerno Feel free to chime in here as there might be a more automated way to do this.

Reply:
Ok, that makes sense! Now the job checks against the hardcoded env value. I added a regex to match the version as `rustc --version` returns a string with other info, like hash etc. 

The CI dependency flow now looks like : 
<img width="1268" alt="Screenshot 2023-02-08 at 12 38 18" src="https://user-images.githubusercontent.com/35300528/217531853-2905a7c9-4578-4b27-b206-ed622b330d6b.png">